kalite/coachreports/api_views.py

```diff
@@ -10,7 +10,7 @@
 from kalite.main.models import ExerciseLog, VideoLog, ContentLog, AttemptLog, UserLogSummary
 from kalite.facility.models import FacilityUser
 from kalite.shared.decorators.auth import require_admin
-from kalite.topic_tools import get_topic_leaves, get_exercise_cache, get_content_cache
+from kalite.topic_tools.content_models import get_topic_contents, get_topic_nodes
```

Comment:
This file is a reasonable example of the slight switch this PR makes in terms of the API of interfacing with content caches.